pkg/version/version.go

```diff
@@ -71,6 +85,46 @@ func Version() (string, error) {
 	return releaseName, nil
 }
 
+// ReleaseArchitecture returns the release image cpu architecture version.
+func ReleaseArchitecture() (string, error) {
+	ri, okRI := os.LookupEnv("OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE")
+	if okRI {
+		logrus.Warnf("Found override for release image (%s). Release Image Architecture is unknown", ri)
+		return "", nil
```

Comment:
Would it make sense to do 
```suggestion
		return "unknown", nil
```
so that the output will be
```
release architecture unknown
```
instead of
```
release architecture
```